compiler/noirc_frontend/src/elaborator/enums.rs

```diff
@@ -1060,8 +1066,16 @@ impl<'elab, 'ctx> MatchCompiler<'elab, 'ctx> {
         location: Location,
     ) -> DefinitionId {
         let name = format!("internal_match_variable_{index}");
-        let kind = DefinitionKind::Local(None);
-        let id = self.elaborator.interner.push_definition(name, false, false, kind, location);
+        let definition_kind = DefinitionKind::Local(None);
+        let not_mutable = false;
+        let not_comptime = false;
```

Comment:
I think it would be clearer if these were named `mutable` and `comptime`, otherwise it reads like "not mutable is false, so mutable must be true".